ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16-periodics.yaml

```diff
@@ -38397,7 +38397,7 @@ periodics:
     ci.openshift.io/generator: prowgen
     job-release: "4.16"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-rollback-nightly-azure-upi-f14
+  name: periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-rollback-nightly-azure-upi-f28
```

Comment:
we had a discussion around nightly job in https://github.com/openshift/release/pull/51287#discussion_r1578760798, wondering the signature problem is not an issue now?

Reply:
ah, good catch! I should have updated the latest image to rc.0 but I forgot. Rolling back from nightly to rc should be okay.

Reply:
Updated.